test/models/query_test.rb

```diff
@@ -1160,43 +1161,49 @@ def test_collection_number_all
 
   def test_collection_number_for_observation
     obs = observations(:detailed_unknown_obs)
-    expect = obs.collection_numbers.sort_by(&:format_name)
-    assert_query(expect, :CollectionNumber, observation: obs.id)
+    expects = obs.collection_numbers.sort_by(&:format_name)
+    assert_query(expects, :CollectionNumber, observation: obs.id)
   end
 
   def test_collection_number_pattern_search
-    expect = CollectionNumber.
-             where(CollectionNumber[:name].matches("%Singer%").
-                   or(CollectionNumber[:number].matches("%Singer%"))).
-             sort_by(&:format_name)
-    assert_query(expect, :CollectionNumber, pattern: "Singer")
+    expects = CollectionNumber.
+              where(CollectionNumber[:name].matches("%Singer%").
```

Comment:
Curious about `
matches("%Singer%")` vs 
`matches(/Singer/)`

Reply:
Good question. I think it can also just be `matches("Singer")`.

Reply:
Nope, it has to be `matches("%Singer%")`. The other two won't work.

The slashes won't work in AR, and the plain quoted string apparently doesn't mean the same thing.